migration/migration/004-schemas.sql

```diff
@@ -1,54 +1,39 @@
-CALL _prom_catalog.execute_everywhere('create_schemas', $ee$ DO $$ BEGIN
-    CREATE SCHEMA IF NOT EXISTS _prom_catalog; -- this will be limited to metric and probably renamed in future
-    GRANT USAGE ON SCHEMA _prom_catalog TO prom_reader;
+-- _prom_catalog is created before
```

Comment:
why are we removing the execute_everywhere?

Reply:
Good question. I'm actually not sure that we should be removing it for all schemas, but the basic problem that I'm trying to work around here is:

- `execute_everywhere` runs both on the local node and on all data nodes
- some of the schemas created in the execute everywhere block have already been created on the local node during extension install (e.g. `_prom_ext`, and `_ps_catalog`
- removing `IF NOT EXISTS` breaks the extension install

We talked about how we might install the extension on data nodes as well, in which case we wouldn't want to have these schemas created in an execute everywhere, so I thought that I would remove the execute everywhere for now.

Reply:
done